moodlecli/main.py

```diff
@@ -245,15 +245,17 @@ def export_grades(source_course_id, bucket_name, key):
 @click.argument('bucket_name')
 @click.argument('key')
 def export_users(source_course_id, bucket_name, key):
-    """Output to JSON the users (and their info) for a given course to s3
-    bucket"""
+    """Collects user data and research ids from moodle, injects
```

Comment:
Sorry, I'll keep harping on this :joy: as I'd like us to get away from using terminology like "research IDs". Let's clean / remove that usage in the new strings here.